INDEX

```diff
@@ -102,6 +102,14 @@
     "official": false,
     "primary_pkg": "mosquitto"
   },
+  "netdata": {
+    "MANIFEST": "netdata.json",
+    "description": "Monitor everything in real-time",
+    "icon": "https://icons.freenas.org/community-icons/netdata.png",
+    "primary_pkg": "netdata",
+    "name": "Netdata",
```

Comment:
We have a property `maintainer` as well which is used to track who is the maintainer of a plugin and users can reach out to the relevant maintainer if there is any issue with his/her plugin. Do you think we should have a pass at all the plugins here and add relevant maintainers if any ? ( Having a pass at all the plugins is of course not related to this PR, just some thoughts )